docs/lcd-server-example.md

```diff
@@ -1,112 +0,0 @@
-1. Put this file in `/etc/systemd/system/secret-lcd.service`
```

Comment:
@dylanschultzie where did this file go? I can't find it on docs.scrt.network

Reply:
@assafmo https://github.com/SecretFoundation/docs/blob/main/docs/lcd-server-example.md

For docs that didn't have pages on build.scrt.network, I left them out on docs.scrt.network as well